.readthedocs.yaml

```
@@ -0,0 +1,34 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+   configuration: manual/sphinx/conf.py
+
+# Optionally build your docs in additional formats such as PDF
+#formats:
+#   - pdf
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+   version: 3.8
+   install:
+   - requirements: manual/sphinx/requirements.txt
+
+
+build:
+  apt_packages:
+    - mpich
+    - libmpich-dev
+    - libfftw3-dev
+    - libnetcdf-dev
+    - libnetcdf-cxx-legacy-dev
```

Maybe prefer the cxx4 version?
```suggestion
    - libnetcdf-c++4-dev
```

---

I updated it in the manual as well :+1: